consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/QbftPayload.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.qbft.payload;
+
+import org.hyperledger.besu.consensus.common.bft.ConsensusRoundIdentifier;
+import org.hyperledger.besu.consensus.common.bft.payload.Payload;
+import org.hyperledger.besu.ethereum.rlp.RLPInput;
+import org.hyperledger.besu.ethereum.rlp.RLPOutput;
+
+public abstract class QbftPayload implements Payload {
+
+  protected void writeConsensusRound(final RLPOutput out) {
```

Comment:
Unit test for this?

Reply:
not sure if this is a cop out, but maybe that's an interop test? I.e. if this was only besu-->besu, then simply roundtripping is enough? Trying to ensure that the RLP encoding uses the right types is hard to achieve.